actions/mysql_show_table_status_action_test.go

```diff
@@ -50,11 +50,20 @@ func TestShowTableStatus(t *testing.T) {
 		t.Logf("Full JSON:\n%s", b)
 
 		var actual [][]interface{}
+
+		// Unmarshals the stream of bytes. Uses the following types:
+		//	bool, for JSON booleans
+		//	float64, for JSON numbers
+		//	string, for JSON strings
+		//	[]interface{}, for JSON arrays
+		//	map[string]interface{}, for JSON objects
+		//	nil for JSON null
```

Comment:
Do we really have to quote documentation there?

Reply:
If we have to... I don't know. But for sure we can.

Reply:
Please remove it: we don't quote documentation for library packages – it is assumed that Go developers know packages they use work, especially standard library packages